sonic-thermalctld/scripts/thermalctld

```diff
@@ -544,6 +560,8 @@ class TemperatureUpdater(logger.Logger):
 
         fvs = swsscommon.FieldValuePairs(
             [('temperature', str(temperature)),
+             ('minimum_temperature', str(minimum_temperature)),
```

Comment:
Will this field display in CLI?

Reply:
Not currently. If there is a requirement from multiple vendors, then I thought I will take up the CLI work as part of another enhancement PR.